code/_core/datum/loadout/rev_new.dm

```diff
@@ -158,8 +158,10 @@
 		/obj/item/clothing/head/hat/skimask/black,
 		/obj/item/clothing/hands/gloves/colored/padded/black,
 		/obj/item/clothing/hands/gloves/colored/padded/black/left,
-		/obj/item/clothing/feet/shoes/black_boots,
-		/obj/item/clothing/feet/shoes/black_boots/left,
+		/obj/item/clothing/feet/socks/knee,
+		/obj/item/clothing/feet/socks/knee,
+		/obj/item/clothing/feet/shoes/winter,
```

Comment:
I'm fine with socks but the boots should be jackboots to match the color scheme.